js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx

```diff
@@ -243,6 +243,10 @@ export const AttributeGeneralSettings = () => {
               />
             </FormGroup>
           )}
+        </>
+      )}
+      {"username" != attributeName && (
```

Comment:
Generally we put the constant value to compare to on the right hand side, as well as using [strict inequality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality) for the operator.

```suggestion
      {attributeName !== "username" && (
```